src/stackdriver-spinner/README.md

```diff
@@ -0,0 +1,9 @@
+# Log Spinner CF App
+
```

Comment:
Can you add some documentation about what this app does, the specific metrics it emits, and how it fits into the full pipeline?

It's too early to say "push these buttons and here's your SLO" but providing context will help us stay fresh